<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>FilterOptionsUI</class>
  <widget class="QWidget" name="FilterOptionsUI" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>360</width>
        <height>327</height>
      </rect>
    </property>
    <layout class="QVBoxLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <item>
        <widget class="QCheckBox" name="cbEnableShortcuts" >
          <property name="text" >
            <string>&amp;Enable Web shortcuts</string>
          </property>
          <property name="whatsThis" stdset="0" >
            <string>&lt;qt>
Enable shortcuts that allow you to quickly search for information on the web. For example, entering the shortcut &lt;b>gg:KDE&lt;/b> will result in a search of the word &lt;b>KDE&lt;/b> on the Google(TM) search engine.
&lt;/qt></string>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QGridLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item row="5" column="0" >
            <widget class="QLabel" name="lbDelimiter" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>5</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text" >
                <string>&amp;Keyword delimiter:</string>
              </property>
              <property name="buddy" stdset="0" >
                <cstring>cmbDelimiter</cstring>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>Choose the delimiter that separates the keyword from the phrase or word to be searched.</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="4" column="0" >
            <widget class="QLabel" name="lbDefaultEngine" >
              <property name="enabled" >
                <bool>true</bool>
              </property>
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>5</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text" >
                <string>Default &amp;search engine:</string>
              </property>
              <property name="buddy" stdset="0" >
                <cstring>cmbDefaultEngine</cstring>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>&lt;qt>
Select the search engine to use for input boxes that provide automatic lookup services when you type in normal words and phrases instead of a URL. To disable this feature select &lt;b>None&lt;/b> from the list.
&lt;/qt></string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="4" column="1" >
            <widget class="KComboBox" name="cmbDefaultEngine" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>5</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>&lt;qt>
Select the search engine to use for input boxes that provide automatic lookup services when you type in normal words and phrases instead of a URL. To disable this feature select &lt;b>None&lt;/b> from the list.
&lt;/qt></string>
              </property>
              <item>
                <property name="text" >
                  <string>None</string>
                </property>
              </item>
            </widget>
          </item>
          <item row="1" column="2" >
            <widget class="QPushButton" name="pbChange" >
              <property name="enabled" >
                <bool>false</bool>
              </property>
              <property name="text" >
                <string>Chan&amp;ge...</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>Modify a search provider.</string>
              </property>
            </widget>
          </item>
          <item row="2" column="2" >
            <widget class="QPushButton" name="pbDelete" >
              <property name="enabled" >
                <bool>false</bool>
              </property>
              <property name="text" >
                <string>De&amp;lete</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>Delete the selected search provider.</string>
              </property>
            </widget>
          </item>
          <item row="0" column="2" >
            <widget class="QPushButton" name="pbNew" >
              <property name="text" >
                <string>&amp;New...</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>Add a search provider.</string>
              </property>
            </widget>
          </item>
          <item rowspan="3" row="3" column="2" colspan="1" >
            <spacer name="spacer1" >
              <property name="sizeHint" >
                <size>
                  <width>21</width>
                  <height>170</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Vertical</enum>
              </property>
            </spacer>
          </item>
          <item rowspan="4" row="0" column="0" colspan="2" >
            <widget class="K3ListView" name="lvSearchProviders" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>5</hsizetype>
                  <vsizetype>5</vsizetype>
                  <horstretch>1</horstretch>
                  <verstretch>1</verstretch>
                </sizepolicy>
              </property>
              <property name="showSortIndicator" >
                <bool>true</bool>
              </property>
              <property name="resizeMode" >
                <enum>AllColumns</enum>
              </property>
              <property name="treeStepSize" >
                <number>0</number>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>List of search providers, their associated shortcuts and whether they shall be listed in menus.</string>
              </property>
              <column>
                <property name="text" >
                  <string>Name</string>
                </property>
                <property name="clickable" >
                  <bool>true</bool>
                </property>
                <property name="resizable" >
                  <bool>true</bool>
                </property>
              </column>
              <column>
                <property name="text" >
                  <string>Shortcuts</string>
                </property>
                <property name="clickable" >
                  <bool>true</bool>
                </property>
                <property name="resizable" >
                  <bool>true</bool>
                </property>
              </column>
            </widget>
          </item>
          <item row="5" column="1" >
            <widget class="KComboBox" name="cmbDelimiter" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>5</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>Choose the delimiter that separates the keyword from the phrase or word to be searched.</string>
              </property>
              <item>
                <property name="text" >
                  <string>Colon</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Space</string>
                </property>
              </item>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <customwidgets>
    <customwidget>
      <class>K3ListView</class>
      <extends>Q3ListView</extends>
      <header>k3listview.h</header>
    </customwidget>
    <customwidget>
      <class>KComboBox</class>
      <extends>QComboBox</extends>
      <header>kcombobox.h</header>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>cbEnableShortcuts</tabstop>
    <tabstop>lvSearchProviders</tabstop>
    <tabstop>pbNew</tabstop>
    <tabstop>pbChange</tabstop>
    <tabstop>pbDelete</tabstop>
    <tabstop>cmbDefaultEngine</tabstop>
    <tabstop>cmbDelimiter</tabstop>
  </tabstops>
</ui>
